wiki/Tournaments/o!RT/2021/en.md

```diff
@@ -0,0 +1,518 @@
+---
+tags:
+  - o!RST 2021
+  - o!RT
+---
+
+# osu! Romania Summer Tournament 2021
+
+![{SHORT_TITLE} logo](img/logo.png)
+
+The **osu! Romania Summer Tournament 2021** (***o!RST 2021***) was a 1v1, double elimination, country-based osu! tournament for Romanian and Moldovan players hosted by ::{ flag=RO }:: [Nice Aesthetics](https://osu.ppy.sh/users/7782553), ::{ flag=RO }:: [FallenAngelD](https://osu.ppy.sh/users/5225626), and ::{ flag=RO }:: [Ahrinity](https://osu.ppy.sh/users/11587229). It was the 3rd instalment of the osu! Romania Tournament.
```

Comment:
```suggestion
The **osu! Romania Summer Tournament 2021** (***o!RST 2021***) was a 1v1, double elimination osu! tournament for Romanian and Moldovan players hosted by ::{ flag=RO }:: [Nice Aesthetics](https://osu.ppy.sh/users/7782553), ::{ flag=RO }:: [FallenAngelD](https://osu.ppy.sh/users/5225626), and ::{ flag=RO }:: [Ahrinity](https://osu.ppy.sh/users/11587229). It was the 3rd instalment of the osu! Romania Tournament.
```

Reply:
(`country-based` implies that the tournament consists of countries facing up against each other as in the World Cup)